andela/ah-the-phoenix-frontend
# #163388112 users can bookmark articles

#### What does this PR do?
It enables a user to bookmark an article
#### Description of Task to be completed?

 - Create the bookmark functionality
 - List the bookmarked articles in the user profile

#### How should this be manually tested?
1. Clone the repo
```
git clone https://github.com/andela/ah-the-phoenix-frontend.git
```
2. Change it into the current directory
```
cd ah-the-phoenix-frontend
```
3. Checkout to the feature branch
```    
git checkout ft-bookmark-articles-163388112 
```
4. Install the required dependencies using 
```
npm install
```
5. Run the server
```
npm start
```
6. Login to the website then click the bookmark button on an article
7. you can check the bookmarked articles at the user profile

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
[#163388112](https://www.pivotaltracker.com/story/show/163388112)

Screenshots (if appropriate)
